packages/gestalt/src/RadioGroupContext.js

```diff
@@ -0,0 +1,24 @@
+// @flow strict
+import { createContext, useContext, type Context } from 'react';
+
+type RadioGroupContextType = {|
+  parentName: string,
+  hasError?: boolean,
```

Comment:
We don't seem to be using this field, and the Hook below doesn't even return it. Can we remove it?